### PR TITLE
Chrome extensions manager

### LINKED ISSUE #209: [Feature Suggestion] Vue DevTools support
It would be awesome to have the ability to install the Vue DevTools extension, it is provided as a Chrome extension so it should be possible to allow it.

Thank you

### PR DESCRIPTION
**Pull request by RomainFrancony**
Added an Chrome extensions manager

extensions manager

## Features

- install any Chrome extension with it's ID or Chrome Store URL
- delete installed Chrome extensions

## Limitation

- only extensions in `crx` format can be installed
- having multiple devtools opened at the same time (one for the `webview` and one for the app itself) breaks devtools extensions, they only work on the devtools of the app itself
- when killing the app (ex: `ctrl+c` on `yarn dev` in console), it cancels any installation or uninstallation of extensions made during the session. I guess it will do the same if the process is killed with the Task Manager
- in order to update an extension, you need to reenter it's ID, it will force to be redownloaded (I wasn't able to test this but it should work)

fix #209

### CODE REVIEW

File: desktop-app/app/main.dev.js [hunk 1]
```diff
@@ -232,6 +231,14 @@ const createWindow = async () => {
     nativeTheme.themeSource = scheme || 'system';
   });
 
+  ipcMain.handle('install-extension', async (event, id) => {
+    return installExtension(id, true);
```
[the remark this answers is not in the document]
Comment (RomainFrancony) [reply desktop-app/app/main.dev.js:235]: I did that at first but then I found `BrowserWindow.getDevToolsExtensions()` which get the currently installed extensions. So I thought "why keep track of extensions in settings if there is already a API for that" 🤷 

As for the app's auto update, is there a way I can test it locally ?

File: desktop-app/app/components/ExtensionsManager/index.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+              type="text"
+              color="secondary"
+              value={extensionId}
+              onChange={handleChange}
+              disabled={loading}
+              placeholder="Extension ID"
+              variant="outlined"
+              error={errorMessage !== ''}
+            />
+            { errorMessage &&
```
[the remark this answers is not in the document]
Comment (RomainFrancony) [reply desktop-app/app/components/ExtensionsManager/index.js:82]: Done !

Capture d’écran 2020-06-30 à 10 52 21

File: desktop-app/app/main.dev.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (manojVivek) [reply desktop-app/app/main.dev.js:235]: @RomainFrancony  We need to publish alpha releases and test it out, I guess. 

Do you want me to release some alpha versions from your branch?

[the remark this answers is not in the document]
Comment (manojVivek) [reply desktop-app/app/main.dev.js:235]: @RomainFrancony Lets just push it as it is and re-visit it we see that behavior.

[the remark this answers is not in the document]
Comment (RomainFrancony) [reply desktop-app/app/main.dev.js:235]: @manojVivek yeah, sorry for the late reply I was trying to find a way to test locally but didn't found any solution 😬

[the remark this answers is not in the document]
Comment (manojVivek) [reply desktop-app/app/main.dev.js:235]: @RomainFrancony No problems, I'll push it live today, let's test it there. 😉

**Pull request merged**